index.php

```diff
@@ -20,7 +20,7 @@
 
 ?>
 
-	<div id="primary" class=" nhsuk-grid-row">
+	<div id="primary" class="clear">
```

Comment:
elegant :) I like it - does removing the `nhsuk-grid-row` not have a knock on effect of changing padding left and right though? If not, we may need to consider changing all of the page layouts to use this...